api/src/main/java/io/strimzi/api/kafka/model/JmxTransSpec.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.JmxTransOutputDefinitionTemplate;
+import io.strimzi.api.kafka.model.template.JmxTransQueryTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Representation for options to be passed into setting up the JmxTrans.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"outputDefinitions", "queries", "resources"})
```

Comment:
it should be "outputDefinitionTemplates"

Reply:
Any reason why specifically "outputDefinitionTemplates", as I have renamed it to `outputDefinitions` below on line 59
```
    @JsonProperty(value = "outputDefinitions", required = true)
     public List<JmxTransOutputDefinitionTemplate> getOutputDefinitionTemplates() {
```